### ISSUE TITLE
Allow loading source from maven central, ~/.m2/ etc

### ISSUE THREAD

**Issue by marshallpierce** [WinterTechForum/skybar#4]
Right now source is only loaded from the filesystem. To be useful in a production environment, we need to be able to load source from source jars in various places. This will also require us to figure out the jar used for a given class so that we can use heuristics to try to find the corresponding source jar.

**Comment by quidryan** [comment]
The core of this issue is to search for source jars. Pulling from Central/.m2 seems related but complicated. It's much easier for users to just include source jars onto the classpath themselves from their build tools. E.g. tasks.war.from(sourceSets.main.allSource). My perception is that if I know I'm running this tool, then I can also include sources.  Not that some sort of resolving/downloading from Central isn't cool tool.

**Comment by marshallpierce** [comment]
Good point. @eirbjo has some repo-browsing code around somewhere, but source jars on the classpath also makes sense.
